### PR TITLE
Add flex redis sample

### PR DESCRIPTION
**Pull request by jonparrott**

### CODE REVIEW

File: appengine/flexible/redis/main.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+redis_password = os.environ.get('REDIS_PASSWORD', None)
+redis_client = redis.StrictRedis(
+    host=redis_host, port=redis_port, password=redis_password)
+# [END client]
+
+
+# [START example]
+@app.route('/')
+def index():
+
```
Comment (waprin) [inline appengine/flexible/redis/main.py]: extra newline?

File: appengine/flexible/redis/main_test.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import main
+import pytest
+
+
+def test_index():
+    try:
+        main.redis_client.set('counter', 0)
```
Comment (waprin) [inline appengine/flexible/redis/main_test.py]: Do we have a context in which this is actually running?

Comment (jonparrott) [inline appengine/flexible/redis/main_test.py]: Yeah I'm about to push a commit to enable it on travis.

**Comment by waprin** [comment]
lgtm

File: appengine/flexible/redis/main.py (hunk 1 above)
Comment (jonparrott) [inline appengine/flexible/redis/main.py]: Fixed.

**Comment by jonparrott** [comment]
@waprin thanks for the swift review. Will merge once the tests for the test god pass.

**Pull request merged**